src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -198,45 +234,46 @@ const OScapNoneOption = ({ setProfileName }) => {
   );
 };
 
-OScapNoneOption.propTypes = {
-  setProfileName: PropTypes.any,
-};
-
-const OScapSelectOption = ({ profile_id, setProfileName, input }) => {
+const OScapSelectOption = ({
+  profile_id,
+  setProfileName,
+  input,
+}: {
+  profile_id: DistributionProfileItem;
+  setProfileName: (name: string) => void;
+  input?: string;
+}) => {
```

Comment:
We usually define a type that holds the info and reference that instead of typing directly.
Could be cool to keep the pattern that is used in the rest of code base. 

```
type OScapSelectOptionPropType = {
  profile_id: DistributionProfileItem;
  setProfileName: (name: string) => void;
  input?: string;
}
```

I agree that's practically equivalent as the other option you wrote, but let's have all the code having the same vibe.